src/core/cri/cri.ts

```diff
@@ -114,7 +112,17 @@ export async function criClient(appPath?: string, port?: number) {
     rawClient: () => client,
 
     open: async (url: string) => {
-      client = await cdp({ port });
+      const maxTries = 5;
+      for (let i = 0; i < maxTries; ++i) {
+        try {
+          client = await cdp({ port });
+        } catch (e) {
+          if (i === maxTries - 1) {
+            throw e;
+          }
+          await sleep(500);
+        }
+      }
```

Comment:
I added this change to solve a sporadic exception I was seeing on Intel x86_64, and it appears to also have solved our M1 crash (!). Seems like we caught a lucky break. I'll take it.

(@dragonstyle, @jjallaire)